SPECS/complianceascode/complianceascode.spec

```diff
@@ -0,0 +1,44 @@
+%global debug_package   %{nil}
+
+Summary:        package for creating security policy content for various platforms
+Name:           complianceascode
+Version:        1.0
```

Comment:
Since this is our own source project we are packaging, please use semantic versioning. https://semver.org/